bundles/org.openhab.binding.wled/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.wled-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
+
+	<feature name="openhab-binding-wled" description="WLED Binding" version="${project.version}">
+		<feature>openhab-runtime-base</feature>
+		<bundle dependency="true">mvn:org.eclipse.paho/org.eclipse.paho.client.mqttv3/1.2.2</bundle>
```

Comment:
I don't see you using this dependency in your pom file. Is it needed?

Reply:
REMOVED: It used to be used, but not anymore.